pkg/core/resources/store/options.go

```diff
@@ -211,6 +211,10 @@ func ListOrdered() ListOptionsFunc {
 	}
 }
 
+func (l *ListOptions) IsCacheable() bool {
+	return l.FilterFunc == nil && l.NamePrefix == "" && l.PageOffset == "" && l.PageSize == 0
```

Comment:
I think we can include `NamePrefix`, `PageOffset`, and `PageSize` in the key as well. So it's not cacheable only when `l.FilterFunc != nil`